packages/server/src/queries/complex/pools/index.ts

```diff
@@ -105,6 +107,19 @@ export async function getPools(
     poolNameByDenom: pool.reserveCoins.map(({ denom }) => denom).join("/"),
   }));
 
+  if (params.denoms) {
+    const query = params.denoms.map((denom) => `=${denom}`).join("|");
+
+    denomPools = search(
+      denomPools,
+      ["coinDenoms"],
+      {
+        query,
+      },
+      0.0 // Exact match
+    );
+  }
+
```

Comment:
Instead of adding `denoms` to `PoolFilterSchema`, didn't @mattupham update the pools search to match denoms explicitly? This may not be necessary is what I was trying to say earlier.

Reply:
But it doesn't work with an exact match, if you search just for "SOL" you'll also get "SOLAR"

Reply:
OK we can keep this then. Should we do a find instead of use `search` then just to be more explicit?

Reply:
Yep, I'll add it so we don't use `search` function

Reply:
Done!